src/components/new-request/search.vue

```diff
@@ -255,7 +255,7 @@
               </p>
             </div>
           </v-col>
-          <v-col v-else cols="12">
+          <v-col v-else-if="!isXproFlow" cols="12">
```

Comment:
Not a big fan of doing it but this resolves issue with showing Colin button

Reply:
(Feel free to add well-named getters if it helps untangle the logic in the template.)

Reply:
@JazzarKarim seems to be working for Restorations: 
![image](https://github.com/bcgov/namerequest/assets/116035339/fd648e67-a7c7-4041-bc6a-89820626984b)

Reply:
I tried it in the latest temp URL. Looks good Ketaki! Thanks a lot 👍